doc/workshops/tutorial/content/examples/tutorial3-fenix.md

```diff
@@ -0,0 +1,99 @@
+
+
+- [TMAP8](https://mooseframework.inl.gov/TMAP8/) (Tritium migration)
+- [Cardinal](https://cardinal.cels.anl.gov)
+
+  - Monte-Carlo Neutronics via [OpenMC](https://openmc.org)
+  - Computational Fluid Dynamics via [NekRS](https://nek5000.mcs.anl.gov)
+
+- [Electromagnetics Module](https://mooseframework.inl.gov/modules/electromagnetics/)
+- [Ray Tracing Module](https://mooseframework.inl.gov/modules/ray_tracing/) (Kinetic Plasma Foundation)
+- [Heat Transfer](https://mooseframework.inl.gov/modules/heat_transfer/index.html)
+- [Solid Mechanics](https://mooseframework.inl.gov/modules/solid_mechanics/index.html)
+- [Theraml Hydraulics](https://mooseframework.inl.gov/modules/thermal_hydraulics/index.html)
+
+!---
+
+# style=margin-top:1em;
+
+# PIC in FENIX
+
+
+
+In [!cite](lieberman1994principles) this tutorial problem is solved with a PIC code. A core part of the FENIX framework is the PIC capabilities that are being developed. As a very basic benchmarking problem the PIC example presented in [!cite](lieberman1994principles) has been replicated in FENIX
+
+
+- 100 Macroparticles are placed on a 100 element mesh
+- Each particle is initially placed exactly in the center of each cell
+- Leap Frog particle stepping is used
+- $\Delta t$ = $1\times 10^{-9} [\text{s}]$
+- Total simulation time of $2.5\times 10^{-8} [\text{s}]$
+- The potential is represented with first order nodal basis functions
+
+# style=margin-top:1em;
+
+!bibtex bibliography
+
+
+!---
+
+# Results Comparison
+
+All figures show FENIX results super imposed on figures 2.2.a, 2.2.b, and 2.2.c from [!cite](lieberman1994principles). Results from [!cite](lieberman1994principles) are in black and grey while FENIX results are in color.
+
+!row!
+
+!media media/lieberman-particles.png
+       style=width:93%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
+       caption=Particle Population Evolution
+
+!media media/lieberman-potential.png
+       style=width:100%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
+       caption=Electrostatic Potential Evolution
+
+!media media/lieberman-vdf.png
+       style=width:100%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
+       caption=Particle Phase Space Evolution
+!row-end!
+
+!---
+
+# Current PIC Status
+
+- Leap frog and Boris particle stepping have been implemented and verified.
+- Electrostatic capabilities have been verified for all first order finite elements in libMesh.
+- Uniform random particle initialization for all first order finite elements in libMesh has been verified.
+
+  - Bounding box particle initialization has also been verified.
+
+- Particle based current density source calculations is near completion and will be verified.
```

Comment:
I would  re-organize a bit around "verified" and "in-development" activities since there is a lot of word repetition here ("has been verified").

Reply:
I separated this into two subsections with lists of what is done and what is under work.